node/src/types/block.rs

```diff
@@ -2173,6 +2208,7 @@ pub(crate) mod json_compatibility {
         proposer: PublicKey,
         deploy_hashes: Vec<DeployHash>,
         transfer_hashes: Vec<DeployHash>,
+        past_finality_signatures: PastFinalitySignatures,
```

Comment:
We might want to convert `PastFinalitySignatures` into actual `BTreeSet<PublicKey>` (or `Vec<PublicKey>`) for the JSON representation - this is external API and it would be good if the values passed via the API were more readable.

This might be a bit hard, though, as we won't be able to just convert a `BlockBody`, then - it will require the set of validators for the past block for context. I'm not sure what the best course of action would be here, we might have to discuss it.

Reply:
Replaced with a vector of bits, as discussed.